notebooks/text-to-image-genai/text-to-image-genai.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    def infer(model_dir, prompt):
</pre><p>this approach is good for sample in docs, but it does not means that it is good for notebooks as you defines inside a lot of additional arguments that can not be modified by user directly without rewriting body of this function. my opinion, it is make unclear from education point of view as user need to go deeper in this function to understand how it works and simple search of examples how pipeline works do not gives answer immediately. So for me, direct usage of pipeline class in notebooks is better:</p><ol><li>build strong connection between provided arguments and API code</li><li>gives opportunity to compare original diffusers API and GenAI</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
fixed